services/web/server/src/simcore_service_webserver/projects/_projects_service.py

```diff
@@ -1584,48 +1580,55 @@ async def _open_project() -> bool:
         return False
 
 
-async def try_close_project_for_user(
-    user_id: int,
-    project_uuid: str,
+async def close_project_for_user(
+    user_id: UserID,
+    project_uuid: ProjectID,
     client_session_id: str,
     app: web.Application,
     simcore_user_agent: str,
+    *,
+    wait_for_service_closed: bool = False,
 ):
     with managed_resource(user_id, client_session_id, app) as user_session:
         current_user_session = user_session.get_id()
         all_user_sessions_with_project = await user_session.find_users_of_resource(
-            app, key=PROJECT_ID_KEY, value=project_uuid
+            app, key=PROJECT_ID_KEY, value=f"{project_uuid}"
         )
 
         # first check whether other sessions registered this project
         if current_user_session not in all_user_sessions_with_project:
             # nothing to do, I do not have this project registered
-            log.warning(
-                "%s is not registered as resource of %s. Skipping close project",
-                f"{project_uuid=}",
-                f"{user_id}",
-                extra=get_log_record_extra(user_id=user_id),
-            )
             return
 
         # remove the project from our list of opened ones
-        with log_context(
-            log, logging.DEBUG, f"removing {user_id=} session for {project_uuid=}"
-        ):
-            await user_session.remove(key=PROJECT_ID_KEY)
+        await user_session.remove(key=PROJECT_ID_KEY)
```

Comment:
https://github.com/ITISFoundation/osparc-simcore/pull/8140#discussion_r2224764704

Reply:
yep. but that will come later. there are many other things.